templates/CRM/Admin/Page/JobLog.tpl

```diff
@@ -19,6 +19,7 @@
 
   <div class="action-link">
     <a href="{crmURL p='civicrm/admin/job' q="reset=1"}" id="jobsList-top" class="button"><span><i class="crm-i fa-chevron-left" aria-hidden="true"></i> {ts}Back to Scheduled Jobs Listing{/ts}</span></a>
+    <a href="{$jobRunUrl}" id="jobsList-run" class="button"><span><i class="crm-i fa-play" aria-hidden="true"></i> {ts}Execute Now{/ts}</span></a>
```

Comment:
The button and the one below should be hidden if you're on the "All jobs" log where `jid` is blank, since then clicking the button crashes.

Also I'd set `id="jobsList-run-top"` and similarly `id="jobsList-run-bottom` for the bottom one.

Reply:
:+1: Added